### PR TITLE
Feature: update neon IDE integration

### PR DESCRIPTION
**Pull request by f3l1x**
I've added a few new integrations, which seems to be mising.

### CODE REVIEW

File: cs/neon.texy [hunk 1]
```diff
@@ -66,7 +66,11 @@ Viz také [NEON for JavaScript |https://github.com/matej21/neon-js] a [NEON for
 Integrace do IDE
 ================
 
-NetBeans má podporu vestavěnou, pro PhpStom existuje [NEON plugin |https://plugins.jetbrains.com/plugin/7060?pr] a pro Visual Studio Code [taktéž |https://marketplace.visualstudio.com/items?itemName=Kasik96.latte].
+- NetBeans (má vestavěnou podporu)
+- PhpStom ([plugin |https://plugins.jetbrains.com/plugin/7060?pr])
```
Comment (dg) [inline cs/neon.texy]: PhpStorm

Comment (f3l1x) [reply cs/neon.texy]: Thanks, good catch. Sorry ;-)

**Comment by f3l1x** [comment]
Drobne typo upraveno, napada te k tomu jeste neco?

**Comment by dg** [comment]
👍

**Comment by f3l1x** [comment]
Thanks.

**Pull request merged**